### PR TITLE
Added batch start and end date validation

### PR DESCRIPTION
**Pull request by ph4ni**
Added validation for batch dates so that the end date will always be later than the start date.

### CODE REVIEW

**Comment by pateljannat** [comment]
@ph4ni please add a description

File: lms/lms/doctype/lms_batch/lms_batch.py [hunk 1]
```diff
@@ -23,6 +23,7 @@ class LMSBatch(Document):
 	def validate(self):
 		if self.seat_count:
 			self.validate_seats_left()
+		self.validate_batch_end_date():
```
Comment (pateljannat) [inline lms/lms/doctype/lms_batch/lms_batch.py:26]: Remove the colon at the end

File: lms/lms/doctype/lms_batch/lms_batch.py [hunk 2]
```diff
@@ -31,6 +32,10 @@ def validate(self):
 		self.send_confirmation_mail()
 		self.validate_evaluation_end_date()
 
+	def validate_batch_end_date(self):
+		if(self.end_date<self.start_date):
```
Comment (pateljannat) [inline lms/lms/doctype/lms_batch/lms_batch.py:36]: Add space beside the operator.

File: lms/lms/doctype/lms_batch/lms_batch.py [hunk 3]
```diff
@@ -31,6 +32,10 @@ def validate(self):
...
+			frappe.throw("Batch end date cannot be before the batch start date")
```
Comment (pateljannat) [inline lms/lms/doctype/lms_batch/lms_batch.py:37]: Make this string translatable.

**Review by pateljannat — CHANGES_REQUESTED** [review]

**Comment by ph4ni** [comment]
done

**Pull request merged**